README.md

```diff
@@ -52,15 +54,44 @@ $ pip3 install insta-scrape
 ```shell
 $ git clone https://github.com/chris-greening/instascrape.git 
 ```
-> and install required dependencies using 
+### Dependencies
+
+> Install required dependencies using 
 ```shell
 $ pip3 install -r requirements.txt
 ```
-
+>Or install these libraries manually: :sleeping:
+```shell
+$ pip3 install beautifulsoup4==4.9.3
+```
+```shell
+$ pip3 install certifi==2020.6.20
+```
+```shell
+$ pip3 install chardet==3.0.4
+```
+```shell
+$ pip3 install idna==2.10
+```
+```shell
+$ pip3 install lxml==4.5.2
+```
+```shell
+$ pip3 install pytest==6.1.1
+```
+```shell
+$ pip3 install requests==2.24.0
+```
+```shell
+$ pip3 install soupsieve==2.0.1
+```
+```shell
+$ pip3 install urllib3==1.25.10
+```
```

Comment:
Hey @beckettnormington, thanks so much for contributing, it looks great! 

Before I merge, could you maybe remove just this one section? I figure emphasizing the requirements.txt as opposed to manually installing each dependency is more maintainable as the dependency versions change. Everything else looks great!

Reply:
Alright, I believe that I have removed those.

Reply:
Awesome work! I'll merge right now, congratulations on your first contribution to open source!!!! ❤️